semgrep-core/src/analyzing/AST_to_IL.ml

```diff
@@ -793,13 +793,102 @@ let rec stmt_aux env st =
       let stmt2 = stmt env stmt2 in
       stmt1 @ stmt2
   | G.DisjStmt _ -> sgrep_construct (G.S st)
+  | G.OtherStmtWithStmt (G.OSWS_With, Some mk_mgr_as_pat, body) ->
```

Comment:
mgr?
manager?

Reply:
That's right, I made it more clear in the code.